go.mod

```diff
@@ -3,4 +3,7 @@ module github.com/tetratelabs/wazero
 // temporarily support go 1.16 per #37
 go 1.16
 
-require github.com/stretchr/testify v1.5.1
+require (
+	github.com/stretchr/testify v1.5.1
+	github.com/twitchyliquid64/golang-asm v0.15.1
```

Comment:
add a comment to remove this later

Reply:
https://github.com/tetratelabs/wazero/pull/60/commits/d160135e0066a9b470dad6d7e22b3f579bb0300b done